agent/statemanager/state_manager.go

```diff
@@ -48,7 +48,10 @@ const (
 	//   b) Add 'ContainerResourcesProvisioned' as a new 'ContainerStatus' enum
 	//   c) Add 'SteadyStateStatus' field to 'Container' struct
 	//   d) Add 'ENIAttachments' struct
-	ECSDataVersion = 6
+	// 7)
```

Comment:
Have we released anything with `6` yet?  I don't know that we need a bump if `6` is unreleased.

Reply:
I'm not sure if this feature will release together with the eni. I will keep an eye on this.

Reply:
The latest released version is 5, so I think we can leave this at 6 unless 6 gets released before this PR is merged.